src/packages/README.md

```diff
@@ -1,6 +1,6 @@
 # Third-party packages
 
-CheriBSD on Morello ships with both hybrid ABI and CheriABI packages
+CheriBSD on Morello ships with hybrid ABI, CheriABI and benchmark ABI  packages
```

Comment:
I wonder if we want to list CheriABI first here and in the table as it's the default ABI and ultimately the point of the exercise.

Reply:
That's a good idea. I was also wondering about that, and I updated the PR to match the priorities in CheriBSD.

Reply:
NB: I’ve usually been capitalising “Benchmark ABI”. Not sure if that’s a good idea though!

Reply:
I'll merge this PR because this change would be out of scope of this PR. We should be capitalising "Hybrid ABI" if we want to use "Benchmark ABI".

I think "Hybrid ABI" and "Benchmark ABI", alongside "CheriABI", make sense and I would approve replacing all occurrences of them in this guide but also man pages and code in CheriBSD. I would be even tempted to drop "the" in "the Hybrid ABI" and "the Benchmark ABI" in sentences to be consistent with CheriABI for which we never use "the".